### PR TITLE
Add Alert demo

### PR DESCRIPTION
**Pull request by Blackbaud-SomaliaJamall**
Per specs outlined here: [https://blackbaud.invisionapp.com/share/T7RPR60C5NG#/screens/359524172](https://blackbaud.invisionapp.com/share/T7RPR60C5NG#/screens/359524172)

### CODE REVIEW

File: src/app/docs/alert/alert-docs.component.ts [hunk 1]
```diff
@@ -2,8 +2,34 @@ import {
...
+  public onDemoSelectionChange(change: SkyDocsDemoControlPanelChange): void {
+    this.demoSettings.closed = false;
+
+    if (change.closable !== undefined) {
+      this.demoSettings.closable = change.closable;
+    }
+    if (change.alertType) {
+      this.demoSettings.type = change.alertType;
+    }
+    console.log(this.demoSettings.closable);
```
Comment (Blackbaud-SteveBrush) [inline src/app/docs/alert/alert-docs.component.ts]: This `console.log` still needed?

File: src/app/public/plugin-resources/code-examples/alert/basic/alert-demo.module.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'sky-alert-demo',
+  templateUrl: './alert-demo.component.html'
+})
+export class SkyAlertDemoComponent {
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/plugin-resources/code-examples/alert/basic/alert-demo.module.ts]: The name of this file is `alert-demo.module.ts`. Is that correct?

Comment (Blackbaud-SteveBrush) [reply src/app/public/plugin-resources/code-examples/alert/basic/alert-demo.module.ts]: There should be a module file and a component file. See: https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/plugin-resources/code-examples/popover/basic/popover-demo.module.ts

Comment (Blackbaud-SomaliaJamall) [reply src/app/public/plugin-resources/code-examples/alert/basic/alert-demo.module.ts]: There is a component and module file? I think everything is correct, let me know if I've missed something

Comment (Blackbaud-SomaliaJamall) [reply src/app/public/plugin-resources/code-examples/alert/basic/alert-demo.module.ts]: Oh I see! I will resolve this thanks!

**Comment by blackbaud-johnly** [comment]
I ported the changes in this PR over to https://github.com/blackbaud/skyux-indicators/pull/54 so that I could add docs to a branch with the demo. I'm pretty sure we were going to need to move the changes over to a branch anyway to get the PR checks to pass. Should we close this PR in favor of the one I set up? Or do we still have changes pending here?

**Comment by Blackbaud-SomaliaJamall** [comment]
@blackbaud-johnly I have one update to push, I will do it this AM and then you can move the changes!

File: src/app/docs/alert/alert-docs.component.html [hunk 3]
```diff
@@ -7,6 +7,45 @@
...
+          [choices]="typeChoices"
+          [initialValue]="'danger'"
+        >
+        </sky-docs-demo-control-panel-radio-group>
+      </sky-docs-demo-control-panel-section>
+
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Closeable"
+          propertyName="closable"
```
Comment (blackbaud-johnly) [inline src/app/docs/alert/alert-docs.component.html]: Shouldn't this property be `closeable`?

File: src/app/docs/alert/alert-docs.component.html [hunk 4]
```diff
@@ -7,6 +7,45 @@
...
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <sky-alert
+      [alertType]="demoSettings.type"
+      [closeable]="demoSettings.closable"
```
Comment (blackbaud-johnly) [inline src/app/docs/alert/alert-docs.component.html]: Shouldn't `closable` at the end be `closeable`?

File: src/app/docs/alert/alert-docs.component.ts [hunk 5]
```diff
@@ -2,8 +2,44 @@ import {
...
+  ];
+
+  public alertMessage = 'Danger alert message';
+
+  public demoSettings: any = {};
+
+  public onDemoSelectionChange(change: SkyDocsDemoControlPanelChange): void {
+    this.demoSettings.closed = false;
+
+    if (change.closable !== undefined) {
```
Comment (blackbaud-johnly) [inline src/app/docs/alert/alert-docs.component.ts]: Same question about `closeable` here. ... I changed `closable` to `closeable` in all instances in my branch after talking to Steve, and everything seems to work fine that way. But let me know if you see a problem with that.

**Comment by blackbaud-johnly** [comment]
Closing in favor of https://github.com/blackbaud/skyux-indicators/pull/54, which pulls in all the demo updates here and also incorporates the docs.

**Pull request closed without merging**